### DISCUSSION TITLE
still a student but my education pack got expired why?

### DISCUSSION THREAD

**Discussion by aayushchugh** [community/community#61448] [GitHub Education]
### Hi everyone,

So I signed up for GitHub education a few months ago my application was approved by GitHub

I am a student till 2025 in the same institute but suddenly I received an email from GitHub saying `Coupon expiration reminder`

<img width="625" alt="image" src="https://github.com/community/community/assets/69336518/32e00bbf-5f9f-48df-b18a-396df19f10a0">

What should I do now??

**Comment by khoadha** [comment]
You should try this 
https://support.github.com/request/education?tags=hubberfy_education

**Reply by aayushchugh** [reply]
I applied again but my request got reject, I used same proof which was accepted earlier
